css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
Make this into a separate rule, the only declaration on the `row` class should be `display: flex`